pkg/tags/tags.go

```diff
@@ -143,3 +156,31 @@ func (ts *Tags) UnmarshalJSON(value []byte) error {
 
 	return err
 }
+
+// GetTagFromStore get a given tag from the state store.
+func (ts *Tags) GetTagFromStore(uid uint32) (*Tag, error) {
+	key := "tags_" + strconv.Itoa(int(uid))
```

Comment:
it would be good to extract the method that creates the key for the tag into a helper method:
```func getTagKey(uid uint32) string {
   return fmt.Sprintf("tags_%d",uid)
}
```
It is also used in `UpdateTag`